src/cli/cli.cpp

```diff
@@ -2635,6 +2635,27 @@ template <> otError Interpreter::Process<Cmd("counters")>(Arg aArgs[])
             {
                 OutputLine("%s: %d", counter.mName, mleCounters->*counter.mValuePtr);
             }
+#if OPENTHREAD_CONFIG_UPTIME_ENABLE
+            struct MleTimeCounterName
+            {
+                const uint64_t otMleCounters::*mValuePtr;
+                const char *                   mName;
+            };
+
+            static const MleTimeCounterName kTimeCounterNames[] = {
+                {&otMleCounters::mDisabledTime, "Time Disabled"}, {&otMleCounters::mDetachedTime, "Time Detached"},
+                {&otMleCounters::mChildTime, "Time Child"},       {&otMleCounters::mRouterTime, "Time Router"},
+                {&otMleCounters::mLeaderTime, "Time Leader"},
+            };
+
+            const uint64_t trackedTime = mleCounters->mTrackedTime;
```

Comment:
Oh yes. The reason of that local variables is that originally I was computing and printing the percentage of time for each role, apart from the time in ms.

But then I removed that lines because the cli command would have outputted information not present in `otMleCounters`. I forgot to remove those variables, thanks 👍